PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -382,6 +382,34 @@ def miniAOD_customizeCommon(process):
         era.toReplaceWith(process.deepTauIDTask,
                           deepTauIDTaskNew_)
 
+    #-- Adding tauID against dead ECal towers to taus
+    process.hpsPFTauDiscriminationByDeadECALElectronRejectionForMiniAOD = \
+        process.hpsPFTauDiscriminationByDeadECALElectronRejection.clone(
+            extrapolateToECalEntrance = True
+        )
```

Comment:
After debugging the discrepancy for UL re-miniAOD in 106X and master, it seems to me that this new module name is not an optimal solution.
- here in 10_6_X it's a bit semantic (the final result is as expected)
- in the master branch we apparently have a wrong behavior: the 94X and UL reminiAODs are silently picking up hpsPFTauDiscriminationByDeadECALElectronRejection from the input file

I think that the more appropriate solution would be to 
- in 106X:
    - remove hpsPFTauDiscriminationByDeadECALElectronRejectionForMiniAOD in 106X
    - add ```run2_miniAOD_devel.toModify(hpsPFTauDiscriminationByDeadECALElectronRejection, extrapolateToECalEntrance = True)``` in RecoTauTag/Configuration/python/HPSPFTaus_cff.py
    - keep injection of the regular ```hpsPFTauDiscriminationByDeadECALElectronRejection``` in the ```makePatTausTask```
- in the master
    - inject ```hpsPFTauDiscriminationByDeadECALElectronRejection``` in the ```makePatTausTask``` for run2_miniAOD_UL and run2_miniAOD_94XFall17

Reply:
Yes, I agree with this comment; this interference between modules included in different eras have been overlooked in this PR. Should be this issue for 106X resolved as part of this PR or as another one? (+ PR for master in both options).

Reply:
>  Should be this issue for 106X resolved as part of this PR or as another one? (+ PR for master in both options).

the latter.
(nothing can practically be done to a PR that would affect the code after the PR is merged)

Reply:
Right! Overlooked/forgotten that this one is merged - too many items to address after holidays :)